packages/gestalt-design-tokens/tokens/color/alias.json

```diff
@@ -3,7 +3,7 @@
     "text": {
       "default": {
         "value": "{color.black.cosmicore.900.value}",
-        "darkValue": "{color.gray.roboflow.50.value}",
+        "darkValue": "{color.white.mochimalist.0.value}",
```

Comment:
These changes also seem unrelated?

Reply:
They're related to the Badge colors, as roboflow 50 is not accessible